utils/ccdatofhir/mapping_datatypes.bal

```diff
@@ -55,20 +55,25 @@ public isolated function mapCcdaIdToFhirIdentifier(xml idElement) returns uscore
         }
     }
 
-    if system is string && extensionVal is string {
+    string? value = ();
```

Comment:
Should we need a new variable? Can't we use extensionVal for the checks?

Reply:
Also, no need of else-if if you're returning in the proceeding if.

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/1c2818b6dff5ac3b00845d83fb3deb5c473f1dd2